### PR TITLE
5. PATCH /api/reviews/:review_id

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -39,4 +39,147 @@ describe("GET /api/categories", () => {
...
+    return request(app)
+      .get("/api/reviews/not_a_number")
+      .expect(400)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("Invalid ID must be a number");
+      });
+  });
+});
+describe("PATCH /api/reviews/:review_id", () => {
+  test("202: returns object with correct keys and values", () => {
```
Comment (Poonam-raj) [inline __tests__/app.test.js]: Patch would be a 200 OK status, 202 indicates a process has not been complete yet or even started yet.

File: __tests__/app.test.js [hunk 2]
```diff
@@ -39,4 +39,147 @@ describe("GET /api/categories", () => {
...
+  test('400: not passed a valid update object', () => {
+    const newVote = { inc_votes: 'not_valid' };
+
+    return request(app)
+      .patch("/api/reviews/invalid_review_id")
+      .send(newVote)
+      .expect(400)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("Must provide a number to update votes");
+      });
```
Comment (Poonam-raj) [inline __tests__/app.test.js:183]: Would also want a test for if the inc_vote key was missing 

e.g. {ink_notes: 2}  or {dog: 5}

**Review by Poonam-raj — CHANGES_REQUESTED** [review]
Looking good, just need to add to the testing and amend the status you're giving to the patch

**Review by HeatherEver — COMMENTED** [review]
Niiiiice ⭐

**Pull request merged**